src/Nox.Generator/Application/Queries/ByIdQueryGenerator.cs

```diff
@@ -24,6 +24,7 @@ public static void Generate(SourceProductionContext context, NoxSolutionCodeGene
                 .WithClassName($"Get{entity.Name}ByIdQuery")
                 .WithFileNamePrefix($"Queries")
                 .WithObject("entity", entity)
+                .WithObject("isVersioned", (entity.Persistence?.IsVersioned ?? true))
```

Comment:
I know we have this on other generator, we would prefer not to have .WithObject and make it clear and obvious on the template {{if entity.Persistence.IsVersioned}}

Reply:
Updated